operations_on_datastructures/linked_list_right_rotation.cpp

```diff
@@ -0,0 +1,215 @@
+/**
+ * @file
+ * @brief Rotate a linked list 'n' number of times.
+ * @details
+ * The linked list is rotated to right side by one step 'n' times.
```

Comment:
Please describe how the algorithm works, a Wikipedia or any other source for algorithm explanation, what the algorithm does, etc..